css/gora.css

```diff
@@ -0,0 +1,540 @@
+	font-size: 30px;	
+}
+
+.subtitle {
+	font-family: 'Quicksand', sans-serif;
+	text-transform: uppercase;
+}
+span {
+	font-size: 14px;
+}
+@media screen and (max-width: 950px) {
+	span {
+		font-size: 20px;
+		
+	}
+}
+.border-bottom {
+	float: left;
+	width: 5%;
+	height: 4px;
+	border-radius: 30px;
+	background-color: #ffa500;
+	position: absolute;
+	margin-top: -35px;
+	margin-left: 10px;
+}
+.nav2 {
+	text-decoration: none;
+}
+.nav-intro2 {
+	line-height: 1.6;
+	font-size: 18px;
+	font-family: 'Cormorant Garamond', serif;
+	display: inline;
+}
+.nav-link2 {
+	color: black;
+	text-decoration: none;
+	margin: 0;
+	padding: 0;
+	width: 100%;
+    height: 100vh;
+}
+/*------------NEXT PAGE-----------*/
+.bg {
+	background: url(../img/03.jpg) no-repeat center;
+	background-size: 100%;
+	background-size:cover;
+	text-align: center;
+	justify-content: center;
+	display: flex;
+}
+.inform {
+	color: white;
+	font-size: 36px;
+	font-family: 'Quicksand', sans-serif;
+	text-transform: uppercase;
+	text-align: center;
+	margin: 0;
+	padding: 0;
+	margin-top: 30px;
+}
+.personal {
+	background: rgba(0, 0, 0, 0.7);
+	text-align: center;
+	justify-content: center;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	display: flex;
+}
+.sub-text {
+	text-align: center;
+	justify-content: center;
+}
+.row {
+	max-width: 100%;
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	text-align: center;
+	color: white;
+	font-family: 'Quicksand', sans-serif;
+	flex-wrap: wrap;
+	margin-top: 100px;
+	margin-bottom: 80px;
+}
+.column {
+	width: 22%;
+	justify-content: space-between;
+	margin: auto;
+}
+@media screen and (max-width: 768px) {
+	.column {
+		width: 45%;
+		text-align: center;
+	}
+}
+@media screen and (max-width: 500px) {
+	.column {
+		width: 90%;
+		justify-content: center;
+		text-align: center;
+	}
+}
+@media screen and (max-width: 900px) {
+	.row {
+		margin-top: 40px;
+	}
+}
+.column-photo {
+	border-radius: 50%;
+	
+}
+.column-text {
+	text-align: center;
+	justify-content: center;
+	flex-wrap: wrap;
+}
+
+.border-inform {
+	display: inline-flex;
+	width: 5%;
+	height: 4px;
+	border-radius: 30px;
+	background-color: #ffa500;
+	text-align: center;
+	justify-content: center;
+}
+.border-intro {
+	display: inline-block;
+	width: 3%;
+	height: 4px;
+	border-radius: 30px;
+	background-color: #ffa500;
+	opacity: 0.4;
+	text-align: center;
+}
+@media screen and (max-width: 900px) {
+	.border-inform {
+		display: none;
+	}
+	.border-intro{
+		display: none;
+	}
+}
+/*------------NEXT PAGE-----------*/
+
+.information {
+	text-align: center;
+	margin: 0;
+	padding: 0;
+	max-width: 100%;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+}
+.title4 {
+	text-align: center;
+	text-transform: uppercase;
+	font-size: 30px;
+	font-family: 'Cormorant Garamond', serif;
+}
+.sub-title4 {
+	font-size: 14px;
+	font-family: 'Playfair Display', serif;
+	margin: 0 auto;
+	text-align: center;
+	max-width: 100%;
+	width: 85%;
+}
+.row2 {
```

Comment:
remove `row2` and use `row` here 
split class `row` into two classes `row` and `section-name_row`